business_requirement_earned_value/__init__.py

```diff
@@ -1,4 +1,5 @@
-# -*- coding: utf-8 -*-
-# © 2017 Elico Corp (https://www.elico-corp.com).
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from . import report
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp

```

Comment:
Previous copyright was OK

Reply:
@Miya609 this one is OK:
`# © 2017 Elico Corp (https://www.elico-corp.com).`
`# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).`